sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/_encode.py

```diff
@@ -24,6 +24,7 @@
     Sequence,
     Collection,
 )
+from typing_extensions import Buffer
```

Comment:
You didnt run in to mypy complaining about Buffer with some of the methods in encode? I had initially done the same but it complained about __get__, even though per the docs it should cover memoryview

Reply:
Nope I hit some warnings in pyamqp but no errors